src/test/config.sol

```diff
@@ -628,7 +628,7 @@ contract Config {
             line:         0,
             dust:         10 * THOUSAND,
             pct:          400,
-            mat:          2430000,  // 24,300%
```

Comment:
Not sure you need to rm this comment

Reply:
It's such a weird number that it the comment helps to indicate that it's intentional

Reply:
done